events/repostream.go

```diff
@@ -11,14 +11,18 @@ import (
 	cid "github.com/ipfs/go-cid"
 )
 
-type LiteStreamHandleFunc func(op repomgr.EventKind, path string, did string, rcid *cid.Cid, rec any) error
+type LiteStreamHandleFunc func(op repomgr.EventKind, seq int64, path string, did string, rcid *cid.Cid, rec any) error
 
 func ConsumeRepoStreamLite(ctx context.Context, con *websocket.Conn, cb LiteStreamHandleFunc) error {
 	return HandleRepoStream(ctx, con, &RepoStreamCallbacks{
 		RepoAppend: func(evt *RepoAppend) error {
+			if evt.TooBig {
+				log.Errorf("skipping too big events for now: %d", evt.Seq)
```

Comment:
More context would be helpful in this log line: at least the repo and commit (which are in the RepoAppend).